app/uk/gov/hmrc/pdsauthcheckerapi/models/UnvalidatedRequest.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pdsauthcheckerapi.models
+
+import play.api.libs.json.{Json, Reads}
+
+case class UnvalidatedRequest(
+                               validityDate: Option[String],
+                               authType: String,
+                               eoris: Seq[String]
+                             )
+
+object UnvalidatedRequest {
+  implicit val reads: Reads[UnvalidatedRequest] = Json.reads
```

Comment:
This just duplicates PdsAuthRequest but without the value type of Eori, it's a regression. Remove it.

Reply:
Yeah this is what I was discussing in the PR description, it does look iffy but this allows us to guarantee that we have the fields we need to validate within the request that is sent to us, as we cant check against the PdsAuthRequest anymore as we are handling the validation within the controller (Soon to be service).